### ISSUE TITLE
Offer a way to hide the original maven output

### ISSUE THREAD

**Issue by OPeyrusse** [mvndaemon/mvnd#407]
As stated in the improvement brought by `mvnd`:
> Improved console output: we believe that the output of a parallel build on a stock Maven is hard to follow. 

Yet, at the end of a successful operation, the whole output is printed. When often relying on mvnd to build massive projects with tens of sub-modules, this results in a flood of terminal windows.
At the same time, the output is often interesting, to look at errors, see warnings.

A partial workaround is to use `mvnd --quiet ...`. It hides most of the output, while still displaying relevant information like errors.
But it remains polluted with INFO or WARNINGS, like some substitutions made by spring boot.

I think that offering an option to redirect the whole maven output to a file, or simply hide it, would be useful.

**Comment by OPeyrusse** [comment]
This may be an improvement for #355 where I am guessing that the user wants to redirect the output to a file to more easily inspect it, rather than scrolling the console.

**Comment by gnodet** [comment]
> I think that offering an option to redirect the whole maven output to a file, or simply hide it, would be useful.

Have you tried the `-l` option ?

**Comment by OPeyrusse** [comment]
Nope. I had not seen it. And testing it this morning is exactly what I wanted.
Many thanks
